auth.py

```diff
@@ -26,3 +28,14 @@ def authenticate(username, password):
             return True
 
         return False
+
+
+def require_login():
+     if not current_user:
+          return True
+     return not current_user.is_authenticated and AUTH_ENABLED
+
+def show_logout_button():
+     if not current_user:
+          return True
```

Comment:
I'm confused – why should logout button be displayed when there is no `current_user`? Can you add unit tests to these two functions to show the expected return value for various scenarios?